### PR TITLE
feat(ui): add pipeline and delete build assets

### LINKED ISSUE #874: [FEATURE] Move UI to top-level folder and add build/release pipeline
Now that wash is in the monorepo, we should:
- pull washboard to a top level folder
- add a build and release pipeline for the washboard assets

### PR DESCRIPTION
**Pull request by lachieh**
## Feature or Problem
Adds pipeline for the washboard ui

> [!IMPORTANT]  
> This should not be merged until the `wash-cli` is updated to pull releases instead of depending on the asset directory. This work is tracked against #875

## Related Issues
Closes #874

## Release Information
`washboard-v0.1.0`

## Consumer Impact
wash-cli will need to be updated to make use of the release assets for this pipeline

## Testing
pipeline building and asset released

### CODE REVIEW

File: .github/workflows/washboard.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: washboard
+
+on:
+  push:
+    paths: 
+      - 'washboard-ui/**'
+    branches:
+      - main
+    tags:
+      - 'washboard-v*'
```
Comment (connorsmith256) [inline .github/workflows/washboard.yml:10]: For consideration: `washboard-ui`?

**Review by connorsmith256 — APPROVED** [review]

File: .github/workflows/washboard.yml (hunk 1 above)
Comment (lachieh) [inline .github/workflows/washboard.yml:10]:
```suggestion
      - 'washboard-ui-v*'
```

**Review by connorsmith256 — APPROVED** [review]

File: .github/workflows/washboard.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+name: washboard
```
Comment (rvolosatovs) [inline .github/workflows/washboard.yml:1]: could you also update `wash.yml` accordingly to remove the UI build?

File: .github/workflows/washboard.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+on:
+  push:
+    paths: 
+      - 'washboard-ui/**'
```
Comment (rvolosatovs) [inline .github/workflows/washboard.yml:6]: Is there a reason not to just use `washboard`?

Comment (lachieh) [reply .github/workflows/washboard.yml:6]: Surfacing the fact that it is a UI, much like `wash` vs `wash-cli`

File: .github/workflows/washboard.yml (hunk 2 above)
Comment (lachieh) [reply .github/workflows/washboard.yml:1]: Was going to, but it seems more related to the work that will be completed in #875

Comment (connorsmith256) [reply .github/workflows/washboard.yml:1]: Yep I removed in https://github.com/wasmCloud/wasmCloud/pull/919

**Pull request merged**